boot/boot_serial/src/boot_serial.c

```diff
@@ -439,6 +439,45 @@ bs_upload(char *buf, int len)
     flash_area_close(fap);
 }
 
+#ifdef MCUBOOT_BOOT_MGMT_ECHO
+static bool 
+decode_echo(cbor_state_t *state, cbor_string_type_t *result)
+{
+    size_t bsstrdecoded;
+
+    if (!map_start_decode(state)) {
+        return false;
+    }
+    if (!multi_decode(2, 2, &bsstrdecoded, (void *)tstrx_decode, state, result, sizeof(cbor_string_type_t))) {
+        return false;
+    }
+    map_end_decode(state);
+    return true;
+}
+
+
+static void
+bs_echo(char *buf, int len)
+{
+    const char *echo_buf="";
+    size_t echo_buflen = strlen(echo_buf);
```

Comment:
No point to initialize `echo_buf` to "' and `strlen` is wasted call.

Reply:
true, fixed